specs/binding-http.spec/src/main/scripts/io/aklivity/zilla/specs/binding/http/streams/network/rfc7540/connection.management/server.sent.end.stream.before.payload/client.rpt

```diff
@@ -0,0 +1,108 @@
+#
```

Comment:
Rename to `server.sent.end.stream.when.response.body.incomplete` ?

Btw, do we also need `server.sent.close.when.response.body.incomplete` to handle the case where the underlying network transport is closed early, while response body is still in flight.

Reply:
We do have it it is called `client.sent.write.close` perhaps we can rename it